### PR TITLE
Updated developer documentation

### PR DESCRIPTION
**Pull request by Onager**
This is mostly just some cleaning up, and moving around of existing content. There's much more cleanup to be done, focussing on specific pages and topics.

### CODE REVIEW

File: docs/sources/developer/Codereview.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Code Review
 
-All code submitted into the plaso project goes through code review. We use the GitHub codereview process, which while not perfect, is good enough for our purposes.
-
-One helpful hint is while you have a code in code review monitor the development mailing list for large changes or new dependencies that may potentially affect your code. Such changes may include code re-factors that change plugin interface while you have a plugin in review. These should be rare but they do happen every now and then.
+All code submitted into the Plaso project goes through code review. We use the 
+GitHub code review tool.
```
Comment (joachimmetz) [inline docs/sources/developer/Codereview.md]: Maybe even shorter, we use github?

File: docs/sources/developer/Dependencies.md [hunk 2]
```diff
@@ -1,48 +1,36 @@
...
 
-* Timesketch: https://github.com/google/timesketch/blob/master/docs/Installation.md
+* [Timesketch](https://github.com/google/timesketch/blob/master/docs/Installation.md) 
 
 ## Prepackaged dependencies
 
 ### Fedora Core
 
-**Note that the instructions in this page assume you are running on Fedora Core 26, 27 or 28. Installing packages from the copr on other versions and/or distributions is not recommended.**
+**Note that these instructions assume you are running on Fedora Core 26, 27
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: Thjis should be FC 28, 29, 30

File: docs/sources/developer/Dependencies.md [hunk 3]
```diff
@@ -1,48 +1,36 @@
...
+the necessary packages for running plaso. GIFT copr provides the following 
+tracks:
 
-* stable; track intended for the "packaged release" of plaso and dependencies;
-* dev; track intended for the "development release" of plaso;
+* stable; track intended for the "packaged release" of Plaso and dependencies;
+* dev; track intended for development of Plaso;
 * testing; track intended for testing newly created packages.
 
-#### Packaged release
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: why did you remove this?

File: docs/sources/developer/Dependencies.md [hunk 4]
```diff
@@ -56,112 +44,94 @@ To install the dependencies run:
...
 ### MacOS
 
-The [l2tbinaries](https://github.com/log2timeline/l2tbinaries) contains the necessary packages for running plaso. l2tbinaries provides the following branches:
+The [l2tbinaries](https://github.com/log2timeline/l2tbinaries) 
+repository contains the necessary packages for running Plaso. l2tbinaries 
+provides the following branches:
 
-* master; branch intended for the "packaged release" of plaso and dependencies;
-* dev; branch intended for the "development release" of plaso;
+* master; branch intended for the "packaged release" of Pplaso and dependencies;
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: fix typo: Pplaso => Plaso

File: docs/sources/developer/Dependencies.md [hunk 5]
```diff
@@ -56,112 +44,94 @@ To install the dependencies run:
...
+[an update script](https://github.com/log2timeline/l2tdevtools/wiki/Update-script)
+ to ease the process of keeping the dependencies up to date.
 
-#### Development release
+#### Development track
 
 To install the development versions of the dependencies run:
 
-```
+```bash
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: why bash?

File: docs/sources/developer/Dependencies.md [hunk 6]
```diff
@@ -56,112 +44,94 @@ To install the dependencies run:
...
 
 To install plaso from the GIFT PPA you'll need to have Ubuntu universe enabled:
 
-```
+```bash
 sudo add-apt-repository universe
 sudo apt-get update
 ```
 
-#### Packaged release
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: why remove this? where has this moved?

**Comment by joachimmetz** [comment]
@Onager please include changes proposed in https://github.com/log2timeline/plaso/pull/2664 and then close that PR

File: docs/sources/developer/Dependencies.md [hunk 7]
```diff
@@ -1,48 +1,36 @@
...
-This page contains detailed instructions on how to build and install dependencies.
+This page contains detailed instructions on how to install dependencies for 
+development.
 
 There are multiple ways to install the dependencies:
 
-* Using [prepackaged dependencies](Dependencies.md#prepackaged-dependencies);
-* [Batch build](Dependencies.md#batch-build) most of the dependencies;
-* [Manual build](Dependencies.md#manual-build) of the dependencies.
+* [Using pre-packaged dependencies](Dependencies.md#prepackaged-dependencies)
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: both writing forms are valid, I opt to stick with 1 since that was prepackaged please revert this change

File: docs/sources/developer/Dependencies.md [hunk 8]
```diff
@@ -1,48 +1,36 @@
...
 
-* Timesketch: https://github.com/google/timesketch/blob/master/docs/Installation.md
+* [Timesketch](https://github.com/google/timesketch/blob/master/docs/Installation.md) 
 
 ## Prepackaged dependencies
 
 ### Fedora Core
 
-**Note that the instructions in this page assume you are running on Fedora Core 26, 27 or 28. Installing packages from the copr on other versions and/or distributions is not recommended.**
+**Note that these instructions assume you are running on Fedora Core 28, 29
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: remove Core in Fedora Core, also see comment about https://github.com/log2timeline/plaso/pull/2664

File: docs/sources/developer/Dependencies.md (hunk 3 above)
Comment (joachimmetz) [reply docs/sources/developer/Dependencies.md]: I see

File: docs/sources/developer/Dependencies.md [hunk 9]
```diff
@@ -56,112 +44,94 @@ To install the dependencies run:
...
-
-```
-PYTHONPATH=. python tools/update.py --preset plaso
-```
+The l2tdevtools project provides 
+[an update script](https://github.com/log2timeline/l2tdevtools/wiki/Update-script)
+ to ease the process of keeping the dependencies up to date.
 
-#### Development release
+#### Development track
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: remove this heading

File: docs/sources/developer/Dependencies.md [hunk 10]
```diff
@@ -56,112 +44,94 @@ To install the dependencies run:
...
-```
-
-To install the dependencies run:
-
-```
-./config/linux/gift_ppa_install.sh
-```
-
-#### Development release
+#### Development track
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: remove this heading

File: docs/sources/developer/Dependencies.md [hunk 11]
```diff
@@ -56,112 +44,94 @@ To install the dependencies run:
...
-To install the release versions of the dependencies run:
-
-```
-set PYTHONPATH=.
-
-C:\Python27\python.exe tools\update.py --preset plaso
-```
-
-#### Development release
+#### Development branch
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: remove this heading

File: docs/sources/developer/Dependencies.md [hunk 12]
```diff
@@ -1,48 +1,36 @@
 # Plaso Dependencies
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: I opt: Plaso development dependencies

and maybe rename the file to "Development-Dependencies.md" or equiv.

File: docs/sources/developer/Dependencies.md (hunk 5 above)
Comment (joachimmetz) [reply docs/sources/developer/Dependencies.md]: ack

File: docs/sources/developer/Dependencies.md [hunk 13]
```diff
@@ -191,13 +162,18 @@ On other platforms run:
...
-Where `BUILD_TARGET` is the build target for your configuration. If you are unable to find the proper build target we do not recommend using this installation method.
+Where `BUILD_TARGET` is the build target for your configuration. If you can't
+identify the proper build target we do not recommend using this installation 
+method.
 
-Successfully built packages will be stored in the build directory, which is `build` by default. You can use your preferred installation method to install them.
+Successfully built packages will be stored in the build directory, which is 
+`build` by default. You can use your preferred installation method to install them.
 
 ## Manual build
```
Comment (joachimmetz) [inline docs/sources/developer/Dependencies.md]: Seeing you're changing this anyway. I opt to remove this

File: docs/sources/developer/Developers-Guide.md [hunk 14]
```diff
@@ -8,30 +8,16 @@
 
 ## Setting up and maintaining your development environment
 
-The first challenge you will encounter is setting up and maintaining your development environment.
+The first challenge you will encounter is setting up and maintaining your 
+development environment.
 
-Start by installing a development release on your platform:
+Start by preparing a development environment:
```
Comment (joachimmetz) [inline docs/sources/developer/Developers-Guide.md]: maybe setting up instead of preparing ?

File: docs/sources/developer/Developers-Guide.md [hunk 15]
```diff
@@ -50,51 +36,74 @@ Overview of the general architecture of Plaso:
...
+and create an issue and leave it unassigned, note that the priority will be who 
+ever wants to work on it.
 
-Before you start writing the code, please review the following:
+Before you start writing  code, please review the following:
 
-* [Style guide](Style-guide.md). All code submitted to the project needs to follow this style guide.
-* [Code review](Codereview.md). All code that is submitted into the project needs to be reviewed by at least one other person.
-* [Adding a new dependency](https://github.com/log2timeline/l2tdocs/blob/master/process/Dependencies.md). If your code requires adding a new dependency please check out these instructions.
+* [Style guide](Style-guide.md). All code submitted to the project needs to
```
Comment (joachimmetz) [inline docs/sources/developer/Developers-Guide.md]: Wasn't this moved to l2tdocs?

File: docs/sources/developer/Developers-Guide.md [hunk 16]
```diff
@@ -50,51 +36,74 @@ Overview of the general architecture of Plaso:
...
 
-Before you start writing the code, please review the following:
+Before you start writing  code, please review the following:
 
-* [Style guide](Style-guide.md). All code submitted to the project needs to follow this style guide.
-* [Code review](Codereview.md). All code that is submitted into the project needs to be reviewed by at least one other person.
-* [Adding a new dependency](https://github.com/log2timeline/l2tdocs/blob/master/process/Dependencies.md). If your code requires adding a new dependency please check out these instructions.
+* [Style guide](Style-guide.md). All code submitted to the project needs to 
+follow this style guide.
+* [Code review](Codereview.md). All code that is submitted into the project is
```
Comment (joachimmetz) [inline docs/sources/developer/Developers-Guide.md]: I thought the idea was to have this in l2tdocs?

File: docs/sources/developer/Codereview.md [hunk 17]
```diff
@@ -1,8 +1,4 @@
 # Code Review
```
Comment (joachimmetz) [inline docs/sources/developer/Codereview.md]: Maybe remove this page? in favor of the one in l2tdocs?

File: docs/sources/developer/Developers-Guide.md [hunk 18]
```diff
@@ -50,51 +36,74 @@ Overview of the general architecture of Plaso:
...
 ### Before you submit your first code review
 
-1. Join the development mailing list: [log2timeline-dev@googlegroups.com](https://groups.google.com/forum/?fromgroups#!forum/log2timeline-dev), we recommend using the same account as step 1
-2. Install the required development tools like pylint, python-mock, sphinx
-3. Make sure to run all the tests inside the Plaso and the dfVFS codebase, and that they successfully complete on your development system
-4. Make sure your development system is set up correctly so that you can develop and test correctly.
-5. Make sure your email address and name are correctly set in git e.g.:
+1. Join the development mailing list: [log2timeline-dev@googlegroups.com](https://groups.google.com/forum/?fromgroups#!forum/log2timeline-dev),
+we recommend using the same account as step 1
+1. Install the required development tools like pylint, python-mock, sphinx
```
Comment (joachimmetz) [inline docs/sources/developer/Developers-Guide.md]: sphinx is not needed, I opt to remove it

File: docs/sources/developer/Developers-Guide.md [hunk 19]
```diff
@@ -50,51 +36,74 @@ Overview of the general architecture of Plaso:
...
+1. Make sure to run all the tests inside the Plaso and the dfVFS codebase, and
```
Comment (joachimmetz) [inline docs/sources/developer/Developers-Guide.md]: remove `and the dfVFS codebase` ?

File: docs/sources/developer/Developers-Guide.md [hunk 20]
```diff
@@ -50,51 +36,74 @@ Overview of the general architecture of Plaso:
...
+If your code requires adding a new dependency please check out these instructions.
 
 ### Before you submit your first code review
 
-1. Join the development mailing list: [log2timeline-dev@googlegroups.com](https://groups.google.com/forum/?fromgroups#!forum/log2timeline-dev), we recommend using the same account as step 1
-2. Install the required development tools like pylint, python-mock, sphinx
-3. Make sure to run all the tests inside the Plaso and the dfVFS codebase, and that they successfully complete on your development system
-4. Make sure your development system is set up correctly so that you can develop and test correctly.
-5. Make sure your email address and name are correctly set in git e.g.:
+1. Join the development mailing list: [log2timeline-dev@googlegroups.com](https://groups.google.com/forum/?fromgroups#!forum/log2timeline-dev),
```
Comment (joachimmetz) [inline docs/sources/developer/Developers-Guide.md]: maybe mention the slack channel as well? you can reference https://github.com/open-source-dfir/slack

File: docs/sources/developer/Developers-Guide.md [hunk 21]
```diff
@@ -50,51 +36,74 @@ Overview of the general architecture of Plaso:
...
+we recommend using the same account as step 1
+1. Install the required development tools like pylint, python-mock, sphinx
+1. Make sure to run all the tests inside the Plaso and the dfVFS codebase, and
+that they successfully complete on your development system
```
Comment (joachimmetz) [inline docs/sources/developer/Developers-Guide.md]: maybe: development system => system

File: docs/sources/developer/Developers-Guide.md [hunk 22]
```diff
@@ -50,51 +36,74 @@ Overview of the general architecture of Plaso:
...
 
-Use `git config -l` to determine the current configuration.
-
+Use `git config -l` to see your current configuration.
 
 ### Core features changes
 
-Sometimes you need to make some change to the core of the Plaso codebase. In those cases we ask that contributors first create a short design doc explaining the rationale behind the change. The design doc needs to contain:
+Sometimes you need to make some change to the core of the Plaso codebase. 
+In those cases we ask that contributors first create a short design doc
```
Comment (joachimmetz) [inline docs/sources/developer/Developers-Guide.md]: design doc => design proposal

File: docs/sources/developer/Developing-MacOS.md [hunk 23]
```diff
@@ -0,0 +1,45 @@
+# Developing on MacOS
+
+## Git
+To download the latest version of Plaso you'll need to install the
```
Comment (joachimmetz) [inline docs/sources/developer/Developing-MacOS.md]: likely easier to get xcode command line utils? If I'm not mistaken that contains git

File: docs/sources/developer/Developing-Ubuntu.md [hunk 24]
```diff
@@ -0,0 +1,120 @@
...
+
+Checkout the Plaso source from the git repo:
+```
+git clone https://github.com/log2timeline/plaso.git
+```
+
+## Dependencies
+
+Install the dependencies, using either:
+* A [virtualenv](Developing-Virtualenv.html#Fedora%20Core)
```
Comment (joachimmetz) [inline docs/sources/developer/Developing-Ubuntu.md]: Link to Ubuntu not Fedora

File: docs/sources/developer/Developing-Ubuntu.md [hunk 25]
```diff
@@ -0,0 +1,120 @@
...
+* The [pre-packaged dependencies](Dependencies.html#Fedora%20Core).
```
Comment (joachimmetz) [inline docs/sources/developer/Developing-Ubuntu.md]: same comment about pre-packaged, please use prepackaged or change everything to pre-packaged

File: docs/sources/developer/Developing-Ubuntu.md [hunk 26]
```diff
@@ -0,0 +1,120 @@
...
+python utils/check_dependencies.py
+```
+
+**Note that some dependencies are actively under development and can be 
+frequently updated, therefore we recommend checking the status of the
+ dependencies regularly.**
+
+## Updating your environment.
+
+To update the Plaso source code, run:
```
Comment (joachimmetz) [inline docs/sources/developer/Developing-Ubuntu.md]: remove this since origin will be a fork in most cases

File: docs/sources/developer/Developing-Ubuntu.md [hunk 27]
```diff
@@ -0,0 +1,120 @@
...
+
+### PyLint
+Currently plaso development uses PyLint version 1.9.x. 
+
+Remove any older version of PyLint.
+```bash
+sudo apt-get remove pylint
+```
+
+For pylint 1.9.x the following additional dependencies are required:
```
Comment (joachimmetz) [inline docs/sources/developer/Developing-Ubuntu.md]: describe installing this from the pylint3 GIFT PPA ?

Comment (joachimmetz) [reply docs/sources/developer/Developing-Ubuntu.md]: maybe remove the PyLint and Python Mock sections and replace this with an explanation about adding pylint3 PPA and installing pylint and mock via apt?

File: docs/sources/developer/Developing-Windows.md [hunk 28]
```diff
@@ -1,55 +1,68 @@
...
+## Dependencies
 
-Check if you have all the dependencies installed and have the right minimum version:
+Install the dependencies, using the 
+[pre-packaged dependencies](Dependencies.html#Windows).
+
+Check if you have all the dependencies installed and have the right minimum 
+version:
 ```
 C:\Python27\python.exe utils\check_dependencies.py
```
Comment (joachimmetz) [inline docs/sources/developer/Developing-Windows.md]: 27 => 37 ?

File: docs/sources/developer/Developing-Windows.md [hunk 29]
```diff
@@ -1,55 +1,68 @@
 # Developing on Windows
 
-To download the latest version of Plaso you'll need to install the git tools: http://git-scm.com/downloads
+## Git
```
Comment (joachimmetz) [inline docs/sources/developer/Developing-Windows.md]: Add text about the preferred python version? 3.7 ?

File: docs/sources/developer/Developing-Windows.md [hunk 30]
```diff
@@ -1,55 +1,68 @@
...
+Plaso source files by setting PYTHONPATH correspondingly.
 ```
 set PYTHONPATH=C:\plaso-build\plaso
 ```
 
-To run e.g. pinfo:
+To run a tool, for example `pinfo`:
 ```
-C:\Python27\python.exe C:\plaso-build\plaso\plaso\frontend\pinfo.py plaso.db
+C:\Python27\python.exe C:\plaso-build\plaso\plaso\frontend\pinfo.py test.plaso
```
Comment (joachimmetz) [inline docs/sources/developer/Developing-Windows.md]: 27 => 37

File: docs/sources/developer/Developing-Windows.md [hunk 31]
```diff
@@ -1,55 +1,68 @@
...
 
 ## Development tools
-If you intend to do development on plaso you'll also need to install some development tools:
+If you intend to do development on plaso you'll also need to install some 
+development tools:
 
 * PyLint
 * Python Mock
 
 ### PyLint
```
Comment (joachimmetz) [inline docs/sources/developer/Developing-Windows.md]: Maybe replace sections PyLint and Python Mock, by instructions on how to install?

File: docs/sources/developer/How-to-write-a-parser.md [hunk 32]
```diff
@@ -1,19 +1,28 @@
...
+This page assumes you have at least a basic understanding of programming in 
+Python and use of git.
 
 ## Format
 
-Before you can write a binary file parser you will need to have a good understanding of the file format. A description of the Safari Cookies.binarycookies format can be found [here](https://github.com/libyal/assorted/blob/master/documentation/Safari%20Cookies.asciidoc).
+Before you can write a binary file parser you will need to have a good 
+understanding of the file format. A description of the 
+Safari Cookies.binarycookies format can be found 
+[here](https://github.com/libyal/assorted/blob/master/documentation/Safari%20Cookies.asciidoc).
```
Comment (joachimmetz) [inline docs/sources/developer/How-to-write-a-parser.md]: replace URL by https://github.com/libyal/dtformats/blob/master/documentation/Safari%20Cookies.asciidoc

File: docs/sources/developer/How-to-write-a-parser.md [hunk 33]
```diff
@@ -1,19 +1,28 @@
...
 
 ## Parsers vs. Plugins
-Before starting work on a parser, check if Plaso already has a parser that handles the underlying format of the file you're parsing. Plaso currently supports plugins for the following file formats:
+Before starting work on a parser, check if Plaso already has a parser that 
+handles the underlying format of the file you're parsing. Plaso currently 
+supports plugins for the following file formats:
 * Bencode
 * Web Browser Cookies
 * ESEDB
```
Comment (joachimmetz) [inline docs/sources/developer/How-to-write-a-parser.md]: add compound zip to this list?

File: docs/sources/developer/Internals.md [hunk 34]
```diff
@@ -1,52 +1,77 @@
...
+collect important information that can be used to both augment parsing and make
+it more accurate. Examples of what the pre-processing operation should collects:
 
 * Timezone information.
-* Enumerate all users and their paths.
+* Names and metadata about all users, like the path to their home directory.
 * Hostname.
-* Default applications, as in default browser, etc.
-* OS specific items that make future processing simpler (as in current control set in registry, code page used, etc.)
+* OS specific items that make future processing simpler (such as current control
```
Comment (joachimmetz) [inline docs/sources/developer/Internals.md]: OS => operating system

File: docs/sources/developer/Internals.md [hunk 35]
```diff
@@ -1,52 +1,77 @@
...
+Each front-end may decide to run all of these roles in a single thread,
+ multi-thread or on multiple computers.
 
 Also see: [Architecture overview](https://docs.google.com/drawings/d/1WzB3rz50Kf89HtGQ0y28ozPCfTvMo_GVTCMpgAOziy8/preview)
 
 ## Preprocessing
 
-This role needs to be run prior to all other processing. The purpose of this role is to go over an image or a mount point and determine which OS it belongs to and collect important information that can be used to both augment parsing and make it more accurate. Examples of what the pre-processing process should collect:
+This operation runs prior to all other processing. The purpose of this operation
+is to examine the source data and determine ine which OS it belongs to and
```
Comment (joachimmetz) [inline docs/sources/developer/Internals.md]: OS => operating system

File: docs/sources/developer/Manual-Dependencies-Fedora-Core.md [hunk 36]
```diff
@@ -1,28 +1,28 @@
-# Building and Installing Dependencies on Fedora Core 
+# Manually Building and Installing Dependencies on Fedora Core
```
Comment (joachimmetz) [inline docs/sources/developer/Manual-Dependencies-Fedora-Core.md]: per chat remove the manual build instructions

File: docs/sources/user/Troubleshooting-installation-issues.md [hunk 37]
```diff
@@ -142,7 +142,7 @@ There are some packages that declare **libewf2** as one of their dependencies. T
 
 See for instance [this issue](https://github.com/log2timeline/plaso/issues/301).
 
-The solution here is to remove the **libewf2** package from the system and replace it by **libewf** and **libewf2** packages from the GIFT PPA.
+The solution here is to remove the **libewf2** package from the system and replace it with **libewf** and **libewf2** packages from the GIFT PPA.
```
Comment (joachimmetz) [inline docs/sources/user/Troubleshooting-installation-issues.md]: remove `and **libewf2**`

File: docs/sources/developer/Dependencies.md (hunk 12 above)
Comment (Onager) [reply docs/sources/developer/Dependencies.md]: Done

File: docs/sources/developer/Developers-Guide.md (hunk 15 above)
Comment (Onager) [reply docs/sources/developer/Developers-Guide.md]: There's a l2t style guide, but there are Plaso specific things as well - tests, and some more naming stuff to come.

File: docs/sources/developer/Developers-Guide.md (hunk 20 above)
Comment (Onager) [reply docs/sources/developer/Developers-Guide.md]: Done

File: docs/sources/developer/Developers-Guide.md (hunk 18 above)
Comment (Onager) [reply docs/sources/developer/Developers-Guide.md]: Done

File: docs/sources/developer/Developers-Guide.md (hunk 19 above)
Comment (Onager) [reply docs/sources/developer/Developers-Guide.md]: Done

File: docs/sources/developer/Developers-Guide.md (hunk 21 above)
Comment (Onager) [reply docs/sources/developer/Developers-Guide.md]: Changed this to "development environment" to match the "setting up" section.

File: docs/sources/developer/Developers-Guide.md (hunk 22 above)
Comment (Onager) [reply docs/sources/developer/Developers-Guide.md]: Done

File: docs/sources/developer/Developing-MacOS.md (hunk 23 above)
Comment (Onager) [reply docs/sources/developer/Developing-MacOS.md]: Probably. I'd like to update this whole section later.

File: docs/sources/developer/Developing-Ubuntu.md (hunk 24 above)
Comment (Onager) [reply docs/sources/developer/Developing-Ubuntu.md]: Done

File: docs/sources/developer/Developing-Ubuntu.md (hunk 25 above)
Comment (Onager) [reply docs/sources/developer/Developing-Ubuntu.md]: Done

File: docs/sources/developer/Developing-Ubuntu.md (hunk 26 above)
Comment (Onager) [reply docs/sources/developer/Developing-Ubuntu.md]: Done

File: docs/sources/developer/Developing-Ubuntu.md (hunk 27 above)
Comment (Onager) [reply docs/sources/developer/Developing-Ubuntu.md]: I'd like to overhaul all these instructions, but I'll do that in a later change.

File: docs/sources/developer/Developing-Windows.md (hunk 29 above)
Comment (Onager) [reply docs/sources/developer/Developing-Windows.md]: I'd like to overhaul all these instructions, but I'll do that in a later change.

File: docs/sources/developer/Developing-Windows.md (hunk 28 above)
Comment (Onager) [reply docs/sources/developer/Developing-Windows.md]: Done

File: docs/sources/developer/Developing-Windows.md (hunk 30 above)
Comment (Onager) [reply docs/sources/developer/Developing-Windows.md]: Done

File: docs/sources/developer/Developing-Windows.md (hunk 31 above)
Comment (Onager) [reply docs/sources/developer/Developing-Windows.md]: I'd like to overhaul all these instructions, but I'll do that in a later change.

File: docs/sources/developer/How-to-write-a-parser.md (hunk 32 above)
Comment (Onager) [reply docs/sources/developer/How-to-write-a-parser.md]: Done

File: docs/sources/developer/How-to-write-a-parser.md (hunk 33 above)
Comment (Onager) [reply docs/sources/developer/How-to-write-a-parser.md]: Done

File: docs/sources/developer/Internals.md (hunk 34 above)
Comment (Onager) [reply docs/sources/developer/Internals.md]: Done

File: docs/sources/developer/Internals.md (hunk 35 above)
Comment (Onager) [reply docs/sources/developer/Internals.md]: Done

File: docs/sources/developer/Manual-Dependencies-Fedora-Core.md (hunk 36 above)
Comment (Onager) [reply docs/sources/developer/Manual-Dependencies-Fedora-Core.md]: Done

File: docs/sources/user/Troubleshooting-installation-issues.md (hunk 37 above)
Comment (Onager) [reply docs/sources/user/Troubleshooting-installation-issues.md]: Done

File: docs/sources/developer/Developers-Guide.md (hunk 15 above)
Comment (joachimmetz) [reply docs/sources/developer/Developers-Guide.md]: ack

File: docs/sources/developer/Developing-MacOS.md (hunk 23 above)
Comment (joachimmetz) [reply docs/sources/developer/Developing-MacOS.md]: ack

File: docs/sources/developer/Developing-Ubuntu.md (hunk 27 above)
Comment (joachimmetz) [reply docs/sources/developer/Developing-Ubuntu.md]: ack

File: docs/sources/developer/Developing-Windows.md (hunk 29 above)
Comment (joachimmetz) [reply docs/sources/developer/Developing-Windows.md]: ack

File: docs/sources/developer/Developing-Windows.md (hunk 31 above)
Comment (joachimmetz) [reply docs/sources/developer/Developing-Windows.md]: ack
